### PR TITLE
Integrated Presenter and Agenda Form

### PR DESCRIPTION
**Pull request by krishk95**
Integrated Presenter and Agenda forms

### CODE REVIEW

**Review by vigneshwarankarunanithi — APPROVED** [review]

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 1]
```diff
@@ -0,0 +1,489 @@
+define({
```
Comment (mishradileep) [inline KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Put controller file file comment

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 2]
```diff
@@ -0,0 +1,489 @@
...
+
+devHeight :null,
+thisCard :null,
+thisCardIndex:null,
+cardFrameRel :null,
+
+frmAgendaPreshow : function(){
+  var self = this;
+  this.addActionToSessionTiles();
+  //this.view.tileOuterCopy.isVisible=false;
```
Comment (mishradileep) [inline KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Remove commented code

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 3]
```diff
@@ -0,0 +1,489 @@
...
+    self.agendaFilter(eventobject);
+  }
+  this.view.filterDBX.onClick=function(eventobject){
+    self.agendaFilter(eventobject);
+  }
+  this.view.filterQuantum.onClick=function(eventobject){
+    self.agendaFilter(eventobject);
+  }
+  this.view.sessionTileAnim.anchorPoint={"x":.5,"y":0};
+  //this.view.sessionTile.tileBGImageKony.anchorPoint={"x":1,"y":.5};
```
Comment (mishradileep) [inline KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: remove commented code

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 4]
```diff
@@ -0,0 +1,489 @@
...
+  
+},
+
+addActionToSessionTiles : function(){
+  var self = this;
+  var sessionTilesChildren = this.view.sessionTiles.widgets();
+  this.view.sessionTile1.onClick = this.frmAgendaSessionSelect;
+  sessionTilesChildren.forEach(function(currentSessionTile){
+		var currentTileId = currentSessionTile.id;
+    	self.view.sessionTiles[currentTileId].onClick = self.frmAgendaSessionSelect;
```
Comment (mishradileep) [inline KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Format properly (indentation issue here)

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 5]
```diff
@@ -0,0 +1,489 @@
...
+  });
+},
+  
+  
+frmAgendaPostshow : function(){
+	this.devHeight=this.view.masterContainer.frame.height; 
+    egLogger("devHeight = "+this.devHeight);
```
Comment (mishradileep) [inline KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Remove egLogger. Use standard Kony logger

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 6]
```diff
@@ -0,0 +1,489 @@
...
+  this.view.sessionTile10.tilebg.skin="agendaTileSkinDBX";
+  this.view.sessionTile10.tileBGImageKony.src="agendatiledbx.png"
+  this.view.sessionTile11.tilebg.skin="agendaTileSkinQuantum";
+  this.view.sessionTile11.tileBGImageKony.src="agendatilequantum.png"
+  this.view.sessionTile12.tilebg.skin="agendaTileSkinDBX";
+  this.view.sessionTile12.tileBGImageKony.src="agendatiledbx.png"
+  this.view.sessionTile13.tilebg.skin="agendaTileSkinKony";
+  this.view.sessionTile13.tileBGImageKony.src="agendatilekony.png"
+  this.view.sessionTile14.tilebg.skin="agendaTileSkinDBX";
+  this.view.sessionTile14.tileBGImageKony.src="agendatiledbx.png"
```
Comment (mishradileep) [inline KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Put semicolon (;) at end of every line.

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 7]
```diff
@@ -0,0 +1,489 @@
...
+  
+},
+
+frmAgendaSessionSelect : function(eventobject){
```
Comment (mishradileep) [inline KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Put function comment.

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 8]
```diff
@@ -0,0 +1,489 @@
...
+  });
+},
+  
+  
+frmAgendaPostshow : function(){
+	this.devHeight=this.view.masterContainer.frame.height; 
+    egLogger("devHeight = "+this.devHeight);
+},
+
+frmAgendaSetAgendaTiles : function(){
```
Comment (mishradileep) [inline KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Put function comment

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 9]
```diff
@@ -0,0 +1,489 @@
...
+  var sessionTilesChildren = this.view.sessionTiles.widgets();
+  this.view.sessionTile1.onClick = this.frmAgendaSessionSelect;
+  sessionTilesChildren.forEach(function(currentSessionTile){
+		var currentTileId = currentSessionTile.id;
+    	self.view.sessionTiles[currentTileId].onClick = self.frmAgendaSessionSelect;
+  });
+},
+  
+  
+frmAgendaPostshow : function(){
```
Comment (mishradileep) [inline KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Put function comment

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 10]
```diff
@@ -0,0 +1,489 @@
...
+  this.view.sessionTileAnim.anchorPoint={"x":.5,"y":0};
+  //this.view.sessionTile.tileBGImageKony.anchorPoint={"x":1,"y":.5};
+  globalPreshow();
+  this.frmAgendaSetAgendaTiles();
+  
+  this.view.feedbackMaster.opacity=0;
+  
+},
+
+addActionToSessionTiles : function(){
```
Comment (mishradileep) [inline KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Function comment

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaControllerActions.js [hunk 11]
```diff
@@ -0,0 +1,10 @@
+define({
+    /*
+      This is an auto generated file and any modifications to it may result in corruption of the action sequence.
+    */
+    /** preShow defined for frmAgenda **/
+    AS_Form_ad02dde63f0e4c11bbea0ffa927943a6: function AS_Form_ad02dde63f0e4c11bbea0ffa927943a6(eventobject) {
+        var self = this;
+        return self.frmAgendaPreshow.call(this);
+    }
+});
```
Comment (mishradileep) [inline KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaControllerActions.js:10]: Put new line at end of file

File: KonyAccelerate/forms/mobile/frmAgenda.sm/CopyFlexContainer0a95fcd86c47e42.json [hunk 12]
```diff
@@ -0,0 +1,59 @@
+{
```
Comment (mishradileep) [inline KonyAccelerate/forms/mobile/frmAgenda.sm/CopyFlexContainer0a95fcd86c47e42.json:1]: Looks copy skin. Should be removed

File: KonyAccelerate/modules/globalMod.js [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+var devWidth=null;
+
+function globalPreshow(){
+  thisForm=kony.application.getCurrentForm();
+  egLoggerPreshow();
+  egLogger("thisForm = "+thisForm.id);
+}
+
+function assignFormEvents(){
+  //thisForm.preShow=function(){frmCardListPreshow();};
```
Comment (mishradileep) [inline KonyAccelerate/modules/globalMod.js:12]: Remove commented code

**Review by mishradileep — COMMENTED** [review]
Found issues with code formatting. Please fix in all the files
Also lot of commented is being pushed. Please remove commented code
Seems lot of copy skin in commit. please check and remove copy skins 
Remove egLogger and use Kony Standard logger

This apply to all the custom code files.

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js (hunk 4 above)
Comment (krishk95) [reply KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: The formatting is done in the latest PR

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js (hunk 9 above)
Comment (krishk95) [reply KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Added the functional comments

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js (hunk 10 above)
Comment (krishk95) [reply KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Added the functional comments

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaControllerActions.js (hunk 11 above)
Comment (krishk95) [reply KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaControllerActions.js:10]: Done

File: KonyAccelerate/modules/globalMod.js (hunk 13 above)
Comment (krishk95) [reply KonyAccelerate/modules/globalMod.js:12]: Will discuss with Ed and will remove after the discussion

File: KonyAccelerate/forms/mobile/frmAgenda.sm/CopyFlexContainer0a95fcd86c47e42.json (hunk 12 above)
Comment (krishk95) [reply KonyAccelerate/forms/mobile/frmAgenda.sm/CopyFlexContainer0a95fcd86c47e42.json:1]: There are a lot of copy skins. Will take up this task in the last

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js (hunk 5 above)
Comment (krishk95) [reply KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Will discuss with Ed and will remove after the discussion

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js (hunk 3 above)
Comment (krishk95) [reply KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Will discuss with Ed and will remove after the discussion

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js (hunk 2 above)
Comment (krishk95) [reply KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Will discuss with Ed and will remove after the discussion

File: KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js (hunk 1 above)
Comment (krishk95) [reply KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js]: Added the controller file comment

**Pull request merged**